examples/next/hwp-preview-wpgraphql/example-app/src/pages/[identifier].js

```diff
@@ -0,0 +1,86 @@
+import CouldNotLoad from "@/components/CouldNotLoad";
+import Single from "@/components/Single";
+import { client } from "@/lib/client";
+import { getAuthString } from "@/utils/getAuthString";
+import { gql } from "@apollo/client";
+
+// We'll render Single component for these types
+const SINGLE_TYPES = ["Post", "Page", "Building"];
+
+// Get contentNode for previews and nodeByUri for normal content
+const GET_CONTENT = gql`
+  query GetSeedNode($id: ID! = 0, $uri: String! = "", $asPreview: Boolean = false) {
+    nodeByUri(uri: $uri) @skip(if: $asPreview) {
```

Comment:
This feels cumbersome. Since we can add URLs to drafts. Is there a way to enable nodeByUri to support `asPreview`.